scripts/exglobal_atmos_sfcanl.sh

```diff
@@ -190,8 +190,8 @@ fi
 
 # Update surface restarts at middle of window
 for n in $(seq 1 ${ntiles}); do
-    ${NCP} "${COM_ATMOS_RESTART_PREV}/${PDY}.${cyc}0000.sfc_data.tile${n}.nc" \
-            "${COM_ATMOS_RESTART}/${PDY}.${cyc}0000.sfcanl_data.tile${n}.nc"
+    ${NCP} "${COM_ATMOS_ANALYSIS}/${PDY}.${cyc}0000.sfc_data.tile${n}.nc" \
```

Comment:
is this dangerous, what if there is no land DA, will there always be land DA?
General question though for @aerorahul, which is the preferred approach (for this and aero) as long as we are updating the RESTART files, can we save the GES somewhere and update the RESTART in place, or do we need to not touch previously written RESTART, write ANL, and make sure the fcst job knows where to look? Basically, sfcanl does it one way, aero does it another, and we should be consistent (this is on me for not asking this until now)

Reply:
good point @CoryMartin-NOAA
I think we can handle this with a conditional on whether Land DA is exercised or not.
Or is that too simple and I missing something?

Reply:
Conditional is fine, so that would mean you would generally prefer to leave the original RESTART untouched and instead copy/link to the analysis? perhaps this is something you and I should discuss more next week in detail?

Reply:
As suggested, I leave the original RESTART untouched and copy the land analysis to SFCANL when available. I added the following in the script:

```
    if [[ -e "${COM_LAND_ANALYSIS}/${PDY}.${cyc}0000.sfc_data.tile${n}.nc" ]]; then
        ${NCP} "${COM_LAND_ANALYSIS}/${PDY}.${cyc}0000.sfc_data.tile${n}.nc" \
               "${COM_ATMOS_RESTART}/${PDY}.${cyc}0000.sfcanl_data.tile${n}.nc"
    fi
```